### DISCUSSION TITLE
Grid rowGutter colGutter

### DISCUSSION THREAD

**Discussion by Gioxxy** [mantinedev/mantine#4674] [Feature requests]
Could be useful to have the ability to set different vertical and horizontal spacing for Grid columns.
For now, I don't know how to set the space between columns to 0 and between rows to a value, except by creating multiple Grids inside a Stack or using a Stack and Groups.

**Comment by rtivital** [comment]
You can apply styles to the component with `className` or `style` props

**Reply by Gioxxy** [reply]
Yes, I could do everything with HTML&CSS and not use Mantine... this is a feature request, not an help request.

I think this could be very useful because no one will ever use Grid with the same vertical and horizontal spacing. So having this option will make things easier.

If I had a little more free time I would have done a PR.

**Reply by rtivital** [reply]
> For now, I don't know how to set the space between columns to 0 and between rows to a value

I'm sharing a temporary solution because you have mentioned that you are not aware of other ways of applying styles.
